src/main/kotlin/frc/team5499/frc2018Kotlin/subsystems/Drivetrain.kt

```diff
@@ -400,16 +414,14 @@ object Drivetrain : Subsystem() {
     }
 
     // misc functions
-    val position: Vector2
-        get() = Position.positionVector
 
     // super class methods
     override fun update() {
-        Position.update(leftDistance, rightDistance, gyroAngle)
+        Position.update(leftDistance, rightDistance, heading.degrees)
     }
 
     override fun reset() {
-        zeroGyro()
```

Comment:
We dont need to reset the gyro anymore?

Reply:
No, since we're using a heading offset. we never need to zero the gyro. we can just set the heading to 0 and the offset is calculated